src/utils/membership.ts

```diff
@@ -80,9 +80,7 @@ export function isJoinedOrNearlyJoined(membership: string): boolean {
 }
 
 /**
- * Try to ensure the user is already in the megolm session before continuing
- * NOTE: this assumes you've just created the room and there's not been an opportunity
- * for other code to run, so we shouldn't miss RoomState.newMember when it comes by.
```

Comment:
Contrary to what this comment says, even if you call this immediately after `createRoom`, there is still a big window for the `newMember` to arrive in -- for example, after creating the room, `createRoom` calls `setDMRoom` which makes a REST request.